gdc/docker-compose.yml

```diff
@@ -92,10 +92,54 @@ services:
     volumes:
       - dynamodb-data:/home/dynamodblocal/data
 
+# ---------- OpenSearch ----------
+  opensearch:
+    image: opensearchproject/opensearch:2.5.0
```

Comment:
Locked to version `2.5.0` since this is currenlty the latest supported version by [Amazon OpenSearch Service](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/what-is.html) whereas [OpenSearch already supports 2.6.0](https://opensearch.org/docs/latest/)

Reply:
But do we know anything about backwards compatbility? If we get version 2.6, is it going to break code that assumes 2.5? It seems like there haven't been any breaking changes since 2.0. https://opensearch.org/docs/latest/breaking-changes/

https://opensearch.org/faq/#q3.28

Reply:
There should be a tag for `opensearch:2` which would always give you the latest version 2. https://hub.docker.com/layers/opensearchproject/opensearch/2/images/sha256-e628ba709c7c397360c673332e5b6d61adfa203d107a929bd0512edb59c08fb2?context=explore